zeppelin-server/src/main/java/org/apache/zeppelin/service/NotebookService.java

```diff
@@ -344,6 +344,42 @@ public boolean runParagraph(String noteId,
     }
   }
 
+  public boolean runNote(String noteId,
```

Comment:
You can use `runAllParagraphs` https://github.com/apache/zeppelin/blob/master/zeppelin-server/src/main/java/org/apache/zeppelin/service/NotebookService.java#L347
Or maybe rename it to `runNote`

Reply:
I modified existing `runAllParagraphs` to return boolean value depends (that will be translated to exit value of the commandline). And added overloaded method to make it run without providing paragraph data. What do you think?